src/stream.js

```diff
@@ -1,7 +1,27 @@
 'use strict';
 
 const logger = require('./logger');
+const groupBy = require('lodash.groupby');
+const AWS = require('aws-sdk');
+const allowedEvents = ['MODIFY', 'INSERT'];
 
 module.exports.handler = async (event) => {
-	logger.debug({event}, 'Event received');
+	const unmarshalledRecords = event.Records
+		.filter(record => allowedEvents.includes(record.eventName))
+		.map(record => AWS.DynamoDB.Converter.unmarshall(record.dynamodb.NewImage));
+
+	const groupedRecords = groupBy(unmarshalledRecords, record => {
+		if (!record.channel || !record.date) {
+			return 'deleteMe';
```

Comment:
It would be good to log the bad events as it's unexpected.

Reply:
That makes sense.

Reply:
I would suggest logging at the `warn` or `error` log level (e.g. `logger.warn()` or `logger.error()`). Once we're done development, we usually set the log level to `info` so the logs aren't spammed with `debug` logging, which tends to be very verbose. If we received a bad event we would want this to show up in the logs even if `debug` logging was hidden.

Reply:
The package we get the `logger` from is `bunyan`? I'll look that up next time.

Reply:
`bunyan` has 6 log levels that each correspond to a method on the `logger` object:

https://github.com/trentm/node-bunyan#levels

The logger also has a current level setting - set here:

https://github.com/jkehres/trivial-slack-bot/blob/master/src/logger.js#L7

Any logging that occurs below the set level is ignored and not output.